spec/models/mixins/tenant_quotas_mixin_spec.rb

```diff
@@ -0,0 +1,44 @@
+describe TenantQuotasMixin do
+  before do
+    Tenant.seed
+  end
+
+  let(:root_tenant) do
+    Tenant.root_tenant
+  end
+
+  let(:super_admin_role)  { FactoryGirl.create(:miq_user_role, :features => MiqProductFeature::SUPER_ADMIN_FEATURE) }
+  let(:tenant_admin_role) { FactoryGirl.create(:miq_user_role, :features => MiqProductFeature::TENANT_ADMIN_FEATURE) }
+
+  let(:tenant_1)   { FactoryGirl.create(:tenant, :parent => root_tenant) }
+  let(:tenant_1_1) { FactoryGirl.create(:tenant, :name => 'Tenant 2', :parent => tenant_1) }
```

Comment:
Please don't specify `:name` since there is a sequence on the factory

Reply:
👍